pkg/kubelet/cm/dra/state/state_checkpoint_test.go

```diff
@@ -203,4 +208,13 @@ func TestCheckpointStateStore(t *testing.T) {
 	checkpointData, err := checkpoint.MarshalCheckpoint()
 	assert.NoError(t, err, "could not Marshal Checkpoint")
 	assert.Equal(t, expectedCheckpoint, string(checkpointData), "expected ClaimInfoState does not equal to restored one")
+
+	// checkpoint state with an empty checkpointName should return an error when trying to save state to checkpoint
+	cs, err = NewCheckpointState(testingDir, "")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I would argue that `NewCheckpointState()` itself should return an error if no checkpoint filename is passed. Feel free to make that change as part of this PR.

Reply:
done in f924bf95df9